### PR TITLE
AP_GPS_UBLOX: add data injection support

### PR DESCRIPTION
**Pull request by meee1**

### CODE REVIEW

**Comment by tridge** [comment]
looks fine, except last_injected_data_ms is unused. I'll assign to Michael for review

File: libraries/AP_GPS/AP_GPS_UBLOX.cpp [hunk 1]
```diff
@@ -1177,3 +1177,15 @@ AP_GPS_UBLOX::_configure_rate(void)
...
+
+void
+AP_GPS_UBLOX::inject_data(uint8_t *data, uint8_t len)
+{
+
+    if (port->txspace() > len) {
+        last_injected_data_ms = AP_HAL::millis();
+        port->write(data, len);
+    } else {
+        Debug("SBF: Not enough TXSPACE");
```
Comment (WickedShell) [inline libraries/AP_GPS/AP_GPS_UBLOX.cpp]: Shouldn't this be u-blox not SBF? Otherwise looks good to me.

**Comment by meee1** [comment]
ive just modified this pull request to fix the unused variable and the SBF/ UBX issue.

**Comment by tridge** [comment]
applied, thanks!

**Pull request closed without merging**